openai_assistant/utils/event_handler.py

```diff
@@ -122,14 +107,15 @@ async def on_text_done(self, text: Text) -> None:
         # After the message has been processed and sent handle the annotations and update the message
         message = self.current_event.data
         await process_thread_message(
-            message_references=self.message_references,
+            message=self.current_message,
             thread_message=message,
             client=self.client
         )
+        await self.current_message.send()
```

Comment:
Are you sure this is necessary? The message has already been sent in on text create, and we've already updated it in the process_thread_message call.